net/nheko/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           qt5 1.0
+
+github.setup        mujx nheko 0.4.3 v
+categories          net chat
+platforms           darwin
+license             GPL-3
+maintainers         {sireesh @scarface-one gmail.com:sireeshkodali1}
```

Comment:
If you're not a MacPorts member, the `sireesh` is wrong. Am I mistaken on this? Github doesn't seem to think you're a project member.

Reply:
You are right, I'm not a macports member. So I should remove my name I guess?

Reply:
Yes. That would have indicated that `sireesh@macports.org` was a valid email address for you, which it isn't.

Reply:
fixed, thanks